website/cue/reference.cue

```diff
@@ -541,7 +541,7 @@ _values: {
 
 	if !Args.required {
 		// `default` sets the default value.
-		default: string | null
+		default: string | null | *""
```

Comment:
There's a few spots where we don't correctly generate/provide a default value for string fields that aren't marked as being required, so it was simpler to ensure that Cue provides a default value of `""` in those cases.